### PR TITLE
Fix/medium guide

### PR DESCRIPTION
**Pull request by SPrinss**

### CODE REVIEW

**Comment by SPrinss** [comment]
might want to add _team member_ to the glossary...

File: medium-guide/readme.md [hunk 1]
```diff
@@ -4,32 +4,28 @@
...
-> **Frequent dispatches from the front of journalistic experimentation.**
-
->New Atoms is an early-stage news, journalism and research organisation with collaboration — both between humans, and humans and computers — at its core.
-
 Signup on [Medium](https://medium.com/) and ask a team member to invite you as an editor.
 
 ## Write and publish
 
-Click on your profile pick and select new story. You can now write your blogpost add images and make up your text. The [Medium Help center](https://help.medium.com/hc/en-us) learns you a lot about how you can make things work.
+Click on your profile picture and select **new story**. You can now write your blogpost, add images and style your text. The [Medium Help center](https://help.medium.com/hc/en-us) is a great resource for getting started and finding help if you get stuck.
```
Comment (bvhme) [inline medium-guide/readme.md]: You can use one `*` for bold

File: medium-guide/readme.md [hunk 2]
```diff
@@ -4,32 +4,28 @@
...
 
-If you've written your awesome post, choose add to publication an then tick the New Atoms channel. Add draft to publication, now others can review your post because its in draft & stories.
+If you've written your awesome post, choose add to publication and then click **New Atoms channel**. Once you click on **add draft to publication** others can review your post in draft & stories.
 
 1.<img src="../images/medium-add-to-publication.png" width="200"> 2.  <img src="../images/medium-add-draft.png" width="200">
 
 ## Edit
 
-If you're asked to review or edit a medium blogpost always comment and not change. The writer should choose to implement comments. When commenting on text just highlight the text where you want to comment on. Click the comment icon <img src="../images/medium-review-icon.png" width="20"> and send.
+If you're asked to review or edit a Medium blogpost always comment -- never change it yourself. The writer should always choose to implement suggestions. When commenting on text just highlight the text where you want to comment on. Click the comment icon <img src="../images/medium-review-icon.png" width="20"> and send.
```
Comment (bvhme) [inline medium-guide/readme.md]: Use a em-dash in place of `--`, also it shouldnt be padded by spaces.

The way you make an `—` is by doing <kbd>alt</kbd> + <kbd>shift</kbd> + <kbd>-</kbd>

**Comment by bvhme** [comment]
How do you get access to the medium publication?

**Pull request merged**